front/app/components/UI/MoreActionsMenu/index.tsx

```diff
@@ -109,8 +109,9 @@ export interface IAction {
 
 export interface Props {
   actions: IAction[];
-  label?: string | JSX.Element;
-  ariaLabel?: string | JSX.Element;
+  // required for a11y
+  label: string | JSX.Element;
+  showLabel?: boolean;
```

Comment:
We had a type error for title on the MoreOptionsIcon (see below). So what I did instead: require the label instead of label + ariaLabel. And use a different prop than label to determine if the label is shown (showLabel). So then we always have a label for a11y. If we only specified ariaLabel in component, we now have a label + showLabel={false}.

Reply:
See a bit lower for examples.